2004-10-24  Daniel Jacobowitz  <dan@debian.org>

	* opncls.c (bfd_close): Return TRUE for BFD_IN_MEMORY.

Index: src/bfd/opncls.c
===================================================================
RCS file: /big/fsf/rsync/src-cvs/src/bfd/opncls.c,v
retrieving revision 1.25
diff -u -p -r1.25 opncls.c
--- src/bfd/opncls.c	10 Oct 2004 13:58:05 -0000	1.25
+++ src/bfd/opncls.c	24 Oct 2004 17:52:53 -0000
@@ -598,7 +598,7 @@ bfd_close (bfd *abfd)
   if (!(abfd->flags & BFD_IN_MEMORY))
     ret = abfd->iovec->bclose (abfd);
   else
-    ret = 0;
+    ret = TRUE;
 
   /* If the file was open for writing and is now executable,
      make it so.  */
